core/profiles/src/main/java/software/amazon/awssdk/profiles/internal/ProfileFileRefresher.java

```diff
@@ -39,6 +39,7 @@ public final class ProfileFileRefresher {
     private static final ProfileFileRefreshRecord EMPTY_REFRESH_RECORD = ProfileFileRefreshRecord.builder()
                                                                                                  .refreshTime(Instant.MIN)
                                                                                                  .build();
+    private static final long STALE_TIME_MS = 1000;
```

Comment:
Can we add `LogCaptor ` to make sure that warning  is no longer logger/
something like https://github.com/aws/aws-sdk-java-v2/blob/f272552bff83a46dbe3cc8fa11718d4157529a42/core/http-auth-aws/src/test/java/software/amazon/awssdk/http/auth/aws/internal/signer/DefaultRequestSignerTest.java#L72-L74

Reply:
Good call - added an assertion that no warnings are logged.